src/main/resources/templates/db_scripts/data-migration/update_voucherNumver_in_voucher_invoice.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE ${myuniversity}_${mymodule}.vouchers
+SET jsonb = jsonb || jsonb_build_object('voucherNumber', jsonb ->> 'voucherNumber');
```

Comment:
we don't need to update vouchers table, only invoice table to get correct voucher number from voucher table

Reply:
cool, I will remove this part